examples/us-enrichment-api/geo-reference/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	us_enrichment "github.com/smartystreets/smartystreets-go-sdk/us-enrichment-api"
+	"github.com/smartystreets/smartystreets-go-sdk/wireup"
+)
+
+func main() {
+	log.SetFlags(log.Ltime | log.Llongfile)
+
+	client := wireup.BuildUSEnrichmentAPIClient(
+		//wireup.WebsiteKeyCredential(os.Getenv("SMARTY_AUTH_WEB"), os.Getenv("SMARTY_AUTH_REFERER")),
+		wireup.SecretKeyCredential(os.Getenv("SMARTY_AUTH_ID"), os.Getenv("SMARTY_AUTH_TOKEN")),
```

Comment:
In the wireup on other examples there is some explanation to the user of the different things that could go in here. I would take a look at those and copy that into here!